addon/components/dropzone-widget/component.js

```diff
@@ -14,7 +14,7 @@ import layout from './template';
  * {{dropzone-widget
  *   preUpload=attrs.preUpload
  *   buildUrl=buildUrl
- *   listeners=dropzoneListeners
+ *   success=attrs.success
```

Comment:
It would be good to document that it's possible to pass any dropzone event into this component.